numpy/lib/_npyio_impl.pyi

```diff
@@ -150,19 +150,21 @@ def save(
 def save(
     file: str | os.PathLike[str] | _SupportsWrite[bytes],
     arr: ArrayLike,
-    allow_pickle: bool,
+    allow_pickle: bool = ...,
```

Comment:
This overload matches the case where `fix_imports` is passed as positional argument, e.g. `save(my_file, my_arr, True, True)`. And since positional parameters can't follow parameters with a default, the `allow_pickle` default should be removed here

```suggestion
    allow_pickle: bool,
```